### DISCUSSION TITLE
[LOCKED] zombieTypeBuffed

### DISCUSSION THREAD

**Discussion by deftesthawk** [sf-inc/sihywtcamd#16] [Ideas]
I'd love a config setting that allows you to change how this works, for example instead of it making all zombies take 1 damage it could reduce the damage you deal to them by a configurable percentage. I'd probably pick 50% maybe 75% less damage, somewhere around that area. Thanks for opening a discussion tab this is exactly what I needed :)

**Comment by charlyb01** [comment]
I tried to keep the mod with the simplest configuration possible, with only YES/NO configs... but I guess I'll need to make it more configurable. That's a good idea, I may add it soon (maybe when I'll have more changes to do to the mod). Thanks a lot for your suggestion, and feel free to share your other ideas =)

**Reply by deftesthawk** [reply]
Yeah. I completely understand trying to keep it simple, as when I see a mod that has really complicated configs I tend to avoid changing it, as I don't want to unbalance the mod. Your mod has nice simple configs and you've listed what they all do which helps a great deal. :)

**Comment by charlyb01** [comment]
This is out! =)
If you have no other suggestions about Zombies buff, you can close this discussion (and maybe open another one about something else)

**Reply by deftesthawk** [reply]
Thank you I’ll make sure to download it and update my mod pack tomorrow then :D

**Reply by deftesthawk** [reply]
I can’t figure out to close it on my phone so I’ll try close it on my pc tomorrow, and add my new idea then :)

**Reply by charlyb01** [reply]
I said close but actually it is "lock". You can find it on the 3 points at the top right of your screen.
Not a problem if you don't lock it. Can't wait for your other ideas =)

**Reply by deftesthawk** [reply]
Just checked it out, it's literally set to the perfect percentage that I was going todo :D 
This does feel better now as before there was no point in using a tool if you didn't have a way to set them on fire or smite. whereas now you at least get something for using a higher-end weapon.

I've pressed the 3 dots but it doesn't show an option to lock the post unfortunately, I looked around the page and can't find any way to do it.

**Reply by charlyb01** [reply]
I'm glad you liked it. I will lock it, don't worry ;)
